<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>License</class>
 <widget class="QDialog" name="License">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>License</string>
  </property>
  <property name="windowIcon">
   <iconset resource="feedIOicons.qrc">
    <normaloff>:/images/feedIO.png</normaloff>:/images/feedIO.png</iconset>
  </property>
  <widget class="QTextBrowser" name="txtBrserLicense">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>274</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="feedIOicons.qrc"/>
 </resources>
 <connections/>
</ui>
